guides/PureScript-Without-Node.md

```diff
@@ -1,4 +1,5 @@
-The PureScript community has standardized on the NodeJS toolset to a certain extent: we use tools like Pulp, Browserify and Gulp, which run on NodeJS and are installed with NPM, and we use Bower for our package management. However, we have some users who would rather not, or cannot, run NodeJS on their machines. In this short post, I'll demonstrate how to use the PureScript compiler directly on the command line, and how to automate some processes using standard tooling.
+
+The PureScript community has standardized on tools like Pulp, Bower, Browserify, Webpack, and RollupJS, all of which use the NodeJS runtime and use NPM as their supported distribution channel. There are users who cannot, or prefer not to, use NodeJS-based tools. PureScript can be used without NodeJS-based tools.
```

Comment:
I don't think we're ready to mention rollup here... I am yet to have it work successfully on a PS project.

Reply:
I've been trying rollup for some PS projects with success. The rollup-plugin-purs thing has bitrotted a bit and needs updating, so the great tree-shaking isn't ready to talk about yet, but vanilla rollup with the rollup-plugin-commonjs and rollup-plugin-node-resolve plugins work fine.